Source/WarpX.cpp

```diff
@@ -2027,58 +2153,42 @@ WarpX::ClearLevel (int lev)
 }
 
 void
-WarpX::AllocLevelData (int lev, const BoxArray& ba, const DistributionMapping& dm)
-{
-    const bool aux_is_nodal = (field_gathering_algo == GatheringAlgo::MomentumConserving);
+WarpX::AllocLevelData (int lev, const BoxArray& ba,
+                       const DistributionMapping& dm) {
+    const bool aux_is_nodal =
+        (field_gathering_algo == GatheringAlgo::MomentumConserving);
 
     const Real* dx = Geom(lev).CellSize();
 
     // Initialize filter before guard cells manager
     // (needs info on length of filter's stencil)
-    if (use_filter)
-    {
+    if (use_filter) {
         InitFilter();
     }
 
     guard_cells.Init(
-        dt[lev],
-        dx,
-        m_do_subcycling,
-        WarpX::use_fdtd_nci_corr,
-        grid_type,
-        do_moving_window,
-        moving_window_dir,
-        WarpX::nox,
-        nox_fft, noy_fft, noz_fft,
-        NCIGodfreyFilter::m_stencil_width,
-        electromagnetic_solver_id,
-        maxLevel(),
-        WarpX::m_v_galilean,
-        WarpX::m_v_comoving,
-        m_safe_guard_cells,
-        WarpX::do_multi_J,
-        WarpX::fft_do_time_averaging,
+        dt[lev], dx, m_do_subcycling, WarpX::use_fdtd_nci_corr, grid_type,
```

Comment:
Here I think that the original version was more readable. The main issue, however, is probably having methods with more than ~10 parameters!

Reply:
Here too I am not sure if there is a way to make this systematic, as one version might look more readable in one place, the other version in another place. I personally like the Ruff linting we use in Python, which enforces one argument per line, except for very short lines, because it makes reviewing code changes easier. 

These might be our only options:
![Screenshot from 2025-03-11 17-22-20](https://github.com/user-attachments/assets/dde19036-775f-4306-ab11-2618925e0413)